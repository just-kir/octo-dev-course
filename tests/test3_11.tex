\documentclass[12pt]{article}

\usepackage{listings}
%\lstloadlanguages{Ruby}
%\lstset{%
%basicstyle=\ttfamily\color{black},
%commentstyle = \ttfamily\color{red},
%keywordstyle=\ttfamily\color{blue},
%stringstyle=\color{orange}}
%\usepackage[dvipsnames]{xcolor}

\usepackage[utf8]{inputenc}
\usepackage[russian]{babel}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{geometry}
\usepackage{graphicx}
\usepackage{braket}
\usepackage{hyperref}
\usepackage{esint}
\geometry{top=2cm} %поле сверху
\geometry{bottom=2cm} %поле снизу
\geometry{left=2cm} %поле справа
\geometry{right=2cm}
\usepackage{graphicx}
\usepackage{wrapfig} % Обтекание рисунков и таблиц текстом

%\newcommand{\ud}{\mathop{\mathrm{{}d}}\mathopen{}}

\begin{document}

\section{Бинарные числа}
Говорят, что плохой программист – это тот, кто считает, что в одном килобайте 1000 байт, а хороший программист – это тот, кто полагает, что в одном километре 1024 метра.

Многим эта шутка понятна, так как все знают, что в процессах, связанных с информатикой и компьютерной техникой, фигурирует множество значений, выражаемых степенью двойки, то есть чисел вида $2^k$, где $k$ – некоторое неотрицательное целое число. Назовем такие числа бинарными. 
Задано целое число N. Требуется определить, является ли оно бинарным.
\subsection{Входные данные}
Входной файл INPUT.TXT содержит единственное целое число N, не превосходящее 10000 по абсолютной величине.
\subsection{Выходные данные}
На экран выведите YES, если заданное число является бинарным, и NO в противном случае.

\section{Золотой песок}
Сотрудники завода по производству золотого песка из воздуха решили поправить свое финансовое положение. Они пробрались на склад завода, где хранился золотой песок трех видов. Один килограмм золотого песка первого вида они смогли бы продать за A1 рублей, второго вида – за A2 рублей, а третьего вида – за A3 рублей. Так получилось, что у сотрудников оказалось с собой только три емкости: первая была рассчитана на B1 килограмм груза, вторая на B2 килограмм, а третья на B3 килограмм. Им надо было заполнить полностью все емкости таким образом, чтобы получить как можно больше денег за весь песок. При заполнении емкостей нельзя смешивать песок разных видов, то есть, в одну емкость помещать более одного вида песка, и заполнять емкости песком так, чтобы один вид песка находился более чем в одной емкости.

Требуется написать программу, которая определяет, за какую сумму предприимчивые сотрудники смогут продать весь песок в случае наилучшего для себя заполнения емкостей песком.
\subsection{Входные данные}
В единственной строке входного файла INPUT.TXT записано 6 натуральных чисел A1, A2, A3, B1, B2, B3, записанных в одной строке через пробел. Все числа не превосходят 100.
\subsection{Выходные данные}
На экран следует вывести инственное целое число – сумму в рублях, которую смогут сотрудники заработать в случае наилучшего для себя заполнения емкостей песком.


\section{Цветочки}
В рождественский вечер на окошке стояло три цветочка, слева на право: герань, крокус и фиалка. Каждое утро Маша вытирала окошко и меняла местами стоящий справа цветок с центральным цветком. А Таня каждый вечер поливала цветочки и меняла местами левый и центральный цветок. Требуется определить порядок цветов ночью по прошествии $k$ дней.


\subsection{Входные данные}
Во входном файле INPUT.TXT содержится натуральное число $k$ – число дней ($K \leq 1000$).
\subsection{Выходные данные}
На экран требуется вывести три латинских буквы: «G», «C» и «V» (заглавные буквы без пробелов), описывающие порядок цветов на окошке по истечении K дней (слева направо). Обозначения: G – герань, C – крокус, V – фиалка.



\end{document}