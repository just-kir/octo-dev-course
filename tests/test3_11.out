\BOOKMARK [1][-]{section.1}{\376\377\004\021\004\070\004\075\004\060\004\100\004\075\004\113\004\065\000\040\004\107\004\070\004\101\004\073\004\060}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\004\022\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{\376\377\004\022\004\113\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.1}% 3
\BOOKMARK [1][-]{section.2}{\376\377\004\027\004\076\004\073\004\076\004\102\004\076\004\071\000\040\004\077\004\065\004\101\004\076\004\072}{}% 4
\BOOKMARK [2][-]{subsection.2.1}{\376\377\004\022\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.2}% 5
\BOOKMARK [2][-]{subsection.2.2}{\376\377\004\022\004\113\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.2}% 6
\BOOKMARK [1][-]{section.3}{\376\377\004\046\004\062\004\065\004\102\004\076\004\107\004\072\004\070}{}% 7
\BOOKMARK [2][-]{subsection.3.1}{\376\377\004\022\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.3}% 8
\BOOKMARK [2][-]{subsection.3.2}{\376\377\004\022\004\113\004\105\004\076\004\064\004\075\004\113\004\065\000\040\004\064\004\060\004\075\004\075\004\113\004\065}{section.3}% 9
